modules/installation-osp-default-kuryr-deployment.adoc

```diff
@@ -16,21 +16,24 @@ Use the following quota to satisfy a default cluster's minimum requirements:
 
 [options="header"]
 |================================
-|Resource              | Value
-|Floating IP addresses | 3 - plus the expected number of Services of LoadBalancer type
-|Ports                 | 1500 - 1 needed per Pod
-|Routers               | 1
-|Subnets               | 250 - 1 needed per Namespace/Project
-|Networks              | 250 - 1 needed per Namespace/Project
-|RAM                   | 112 GB
-|vCPUs                 | 28
-|Volume storage        | 175 GB
-|Instances             | 7
-|Security groups       | 250 - 1 needed per Service and per NetworkPolicy
-|Security group rules  | 1000
-|Swift containers      | 2
-|Swift objects         | 1
-|Swift available space | 10 MB or more
+|Resource                | Value
```

Comment:
This section will be overwritten by my in-flight changes later, anyway. I'll need to make sure to keep the LB rows. :)